parsers/jscript.c

```diff
@@ -1941,6 +1952,17 @@ static void initialize (const langType language)
 {
 	Assert (ARRAY_SIZE (JsKinds) == JSTAG_COUNT);
 	Lang_js = language;
+
+	TokenPool = objPoolNew (16, (objPoolCreateFunc)newPoolToken,
+		(objPoolDeleteFunc)deletePoolToken, (objPoolClearFunc)clearPoolToken);
```

Comment:
I mean that e.g. `newPoolToken()` could be `static void *newPoolToken(void)` (`objPoolCreateFunc`), and then avoid the cast here.  So, if the function signatures expected by `objPoolNew()` ever change and someone forgets to update that, a compiler could spot it.